nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -93,7 +107,7 @@ def go_idle(self):
     def goal_cb(self, goal):
         # TODO execute subtasks in order and report status
         # Nav -> Triage -> Nav -> Stock -> Nav ->
-        #   Handoff -> Idle
+        #   Handoff -> Home -> Idle
```

Comment:
The Idle task cant be in charge of driving the robot home. The object for driving the robot somewhere is tied to mission_planner and the IdleTask doesnt see it. What does the IDLE task do? should we delete it?

Reply:
I'm not sure what you mean by the object for driving the robot. Is that the room runner action client? If it is, can't both the navigate task and idle task classes have an instance of the client? Since the tasks are blocking, only one client will send an action goal at a time

Reply:
yeah they could but i dont think they should. i think its best for those Task types to be single instace